docs/network.mdx

```diff
@@ -2,41 +2,24 @@
 id: network
 title: Network Information
 description: Detailed information about chains, RPCs, block explorer and indexer
-sidebar_label: Network
+sidebar_label: Network Information
 ---
 
 import {AddSapphireToMetaMask as S, AddSapphireTestnetToMetaMask as ST} from '@site/src/AddToMetaMask';
 
 # Network information
 
-### Mainnet
+## Networks
 
-- Network name: `sapphire`
-- Long network name: `Oasis Sapphire`
-- Chain ID:
-  - Hex: `0x5afe`
-  - Decimal: `23294`
+|                   | Mainnet                             | Testnet                             | Localnet                                   |
+| ------------------| ----------------------------------- | ----------------------------------- | ------------------------------------------ |
+| Network name      | `sapphire`                          | `sapphire-testnet`                  | `sapphire-localnet`                        |
+| Long network name | `Oasis Sapphire`                    | `Oasis Sapphire Testnet`            | `Localnet`                                 |
```

Comment:
Is there a reason for not calling the local chain `Oasis Sapphire Localnet`?

```suggestion
| Long network name | `Oasis Sapphire`                    | `Oasis Sapphire Testnet`            | `Oasis Sapphire Localnet`                                 |
```

Reply:
I do like "Sapphire Localnet" which we have in the web3 gateway [README](https://github.com/oasisprotocol/oasis-web3-gateway/blob/c41077a8cf8ef93d6d16fe6124e38252c058d86a/docker/README.md?plain=1#L58).